www/class/CentreonLDAPAdmin.class.php

```diff
@@ -106,12 +129,20 @@ public function getLdapParameters()
      */
     protected function updateLdapServers($arId)
     {
-        $this->db->query("DELETE FROM auth_ressource_host WHERE auth_ressource_id = " . $this->db->escape($arId));
-        if (isset($_REQUEST['address'])) {
-            $addressList = $_REQUEST['address'] ?? null;
-            $portList = $_REQUEST['port'] ?? null;
-            $sslList = $_REQUEST['ssl'] ?? null;
-            $tlsList = $_REQUEST['tls'] ?? null;
+        $statement = $this->db->prepare("DELETE FROM auth_ressource_host WHERE auth_ressource_id = :auth_ressource_id");
+        $statement->bindValue(':auth_ressource_id', $arId, \PDO::PARAM_INT);
+        $statement->execute();
+        if (isset($_REQUEST['address']) && is_array($_REQUEST['address'])) {
+            $addressList = $this->sanitizeInputArray($_REQUEST['address']);
+            $portList = isset($_REQUEST['port'])
```

Comment:
you removed my suggestions :( (with `is_array` check)

Reply:
I already have taken into account your suggestion && is_array($_REQUEST['address'])